components/speedreader/rust/lib/src/readability/src/extractor.rs

```diff
@@ -285,11 +287,26 @@ pub fn extract_dom(
 
             // Our CSS formats based on id="article".
             dom::set_attr("id", "article", body.clone(), true);
+            dom::set_attr("hidden", "true", body.clone(), true);
             body.to_string()
         }
         _ => top_candidate.to_string(),
     };
 
+    for node in meta.preserved_meta.iter() {
+        if let Some(data) = node.as_element() {
+            let attributes = data.attributes.borrow();
+
+            let mut val: String = String::from("<meta ");
+            for attr in attributes.map.iter() {
+                val =
+                    [val, format!(" {}=\"{}\" ", attr.0.local, attr.1.value.to_string())].concat();
```

Comment:
`std::slice::concat` is an unstable api method. Please put the initial `val` inside the format macro, or use `+=` or `push_str()` like you do at the end of the loop. that will be shorter and easier to read too.

Reply:
@rillian you may consider adding this to https://github.com/brave/security-action; @thypon can help with the rule writing